apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -71,7 +71,7 @@ class ScriptEditor extends React.Component {
     initialWrapupVideo: PropTypes.string,
     initialProjectWidgetVisible: PropTypes.bool,
     initialProjectWidgetTypes: PropTypes.arrayOf(PropTypes.string),
-    initialTeacherResources: PropTypes.arrayOf(resourceShape).isRequired,
+    initialTeacherResources: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
If you are going to keep supporting both data types on the same field, I would suggest defining both shapes, and using PropTypes.oneOf to combine them.